test/attr_comparable_test.rb

```diff
@@ -97,4 +97,50 @@ def initialize(last_name, first_name)
       assert @d2 != @d3
     end
   end
+
+  describe "parameters that are incompatible to order" do
+    before do
+      @d1 = ComparableTestOneParameter.new(false)
+      @d2 = ComparableTestOneParameter.new(true)
+      @d3 = ComparableTestManyParameters.new(false, 'D')
+      @d4 = ComparableTestManyParameters.new(true, 'D')
+      @d5 = ComparableTestManyParameters.new('Kelly', false)
+      @d6 = ComparableTestManyParameters.new('Kelly', true)
+    end
+
+    # Objects with attributes that can't be compared should return nil
```

Comment:
What's this comment here for? If it's part of the test, can it be in the test name?